server/index.js

```diff
@@ -0,0 +1,38 @@
+// connect express
+const express = require('express');
+
+const app = express();
+
+const path = require('path');
+
+const port = 7777;
+
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
+
+// serve client files
+app.use(express.static(path.join(__dirname, '/../client')));
+
+// import DB
+const db = require('../database');
+
+// GET request
+app.get('/booking', (req, res) => {
+  db.find((data) => {
+    res.send(data);
+  });
+});
+
+
+// POST request
+app.post('/booking', (req, res) => {
+  db.update(req.body);
```

Comment:
what if the db doesn't successfully update with the info? Maybe add a callback if it errors out, that includes sending back a 404 instead of a 200?

Reply:
Added a callback to handle an error! Thanks for the suggestion!